docker-compose.yml

```diff
@@ -105,11 +106,8 @@ services:
     depends_on:
       - formsg
     environment:
-      - SERVICES=s3
+      - SERVICES=s3,sqs
       - DATA_DIR=/tmp/localstack/data
-      - ATTACHMENT_S3_BUCKET=local-attachment-bucket
-      - IMAGE_S3_BUCKET=local-image-bucket
-      - LOGO_S3_BUCKET=local-logo-bucket
```

Comment:
These 3 lines were deleted but never re-introduced. Were the env vars unused back then?

Reply:
it used to be initiated in `docker-entrypoint-initaws.d/init-localstack` so docker needed to know it. Now it's init directly in a shell script